sdcm/scylla_bench_thread.py

```diff
@@ -56,6 +69,34 @@ def run(self):
                         event.add_info(node=self.node, line=line, line_number=line_number).publish()
 
 
+class ScyllaBenchTimeSeriesWriteTimestampStartSearcher(FileFollowerThread):
+    def __init__(self, node, sb_mode, sb_workload, sb_log_filename):
+        super().__init__()
+        self.sb_log_filename = sb_log_filename
+        self.parent_cluster = node.parent_cluster
+        self.sb_mode = sb_mode
+        self.sb_workload = sb_workload
+        self.pattern = re.compile(r"Start timestamp:\s+(\d+)")
+
+    def run(self):
+        if self.sb_mode != ScyllaBenchModes.WRITE.value and self.sb_workload != ScyllaBenchWorkloads.TIMESERIES.value:
+            self.stop()
+        while not self.stopped():
```

Comment:
Maybe need to set timeout here

Reply:
I think it is not needed, because this file follower only need for timeseries write